src/debugger/scriptImporter.ts

```diff
@@ -71,7 +71,7 @@ export class ScriptImporter {
      * Writes the script file to the project temporary location.
      */
     private writeAppScript(scriptBody: string, scriptUrl: url.Url): Q.Promise<String> {
-        let scriptFilePath = path.join(this.sourcesStoragePath, scriptUrl.pathname); // scriptFilePath = "$TMPDIR/index.ios.bundle"
+        let scriptFilePath = path.join(this.sourcesStoragePath, path.basename(scriptUrl.pathname)); // scriptFilePath = "$TMPDIR/index.ios.bundle"
```

Comment:
Just to make sure I understand correctly, these two `path.basename` changes are so if the packager gets a request for `http://localhost/foo/index.bundle.ios` it will search in `.vscode/.react/index.bundle.ios` and not `.vscode/.react/foo/index.bundle.ios`, is that right?

If that is correct, then while this wouldn't work for multiple apps running concurrently, it should work for running one app at a time I think, so it looks ok to me.